packages/api/src/routes/medical/internal-mpi.ts

```diff
@@ -42,7 +44,17 @@ router.get(
         lastNameInitial,
       },
     });
-    return res.status(status.OK).json(foundPatients.map(dtoFromModel));
+    const patientsWithUniqueId = foundPatients.map(patient => {
+      const uniqueId = createPatientUniqueId(patient.cxId, patient.id);
```

Comment:
we need to use the id made of the cxID and patient ID in the CQ response because we store the metadata in s3